ci-operator/config/ViaQ/log-file-metric-exporter/ViaQ-log-file-metric-exporter-main.yaml

```diff
@@ -0,0 +1,44 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.7
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7
+  to: log-file-metric-exporter
+promotion:
+  name: "5.2"
+  namespace: logging
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
```

Comment:
This section can be replaced with a 'released' version in order to accelerate testing.   It may be that it only applies to e2e tests but there is no reason AFAIK this component needs to initially be tested against the latest OCP components.  Ref: https://docs.ci.openshift.org/docs/architecture/ci-operator/#testing-with-an-existing-openshift-release  You may want to reach out to #forum-testplatform for clarity as I'm not exactly certain if you need all the children here

Reply:
done